.github/actions/workflow-build/action.yml

```diff
@@ -95,7 +95,8 @@ runs:
         echo "::endgroup::"
 
     - name: Upload artifacts
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
       with:
         name: workflow
         path: workflow/
+        compression-level: 0
```

Comment:
question: Is there a reason not to leave this at the default value?

Reply:
Yeah, `compression-level: 0` is [_drastically_ faster](https://github.com/rapidsai/ops/issues/2982#issuecomment-1856250873) than even `compression-level: 1` (or the default of 6) for larger files. I don't know how big this artifact gets, but considering GHA artifact storage is free for public repos and these are ephemeral logs, I err on the side of speed.

Reply:
Good to know!